core/eolearn/core/utilities.py

```diff
@@ -515,3 +516,28 @@ def constant_pad(X, multiple_of, up_down_rule='even', left_right_rule='even', pa
 def bgr_to_rgb(bgr):
     """Converts Blue, Green, Red to Red, Green, Blue."""
     return bgr[..., [2, 1, 0]]
+
+
+def renamed_and_deprecated(deprecated_class):
+    """ A class decorator that signals that the class has been renamed when initialized
+
+        Example of use:
+
+        .. code-block:: python
+            @renamed_and_deprecated
+            class OldNameForClass(NewNameForClass):
+                ''' Deprecated version of `NewNameForClass`
+                '''
+
```

Comment:
Documentation building reports an issue:

```
WARNING: Error in "code-block" directive:
maximum 1 argument(s) allowed, 10 supplied.

.. code-block:: python
    @renamed_and_deprecated
    class OldNameForClass(NewNameForClass):
        ''' Deprecated version of `NewNameForClass`
        '''
```

I'm not exactly sure but I suggest rechecking indentations and empty lines. Also I would change `'''` -> `"""`.

Reply:
I'll recheck what went wrong. It's very likely missing a newline at the beginning (especially judging by the error).

I went with `'''` because otherwise it clashes with the docstring delimiters.